src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -1749,7 +1750,9 @@ int32_t SystemNative_GetSockOpt(
                 value = value == 0 ? 1 : 0;
             }
             *(int32_t*)optionValue = value;
-
+#else
+            return Error_ENOTSUP;
+#endif
```

Comment:
Should this else block be moved down a line so that the return Error_SUCCESS is inside the ifdef?  I realize it doesn't actually matter, but it feels cleaner.